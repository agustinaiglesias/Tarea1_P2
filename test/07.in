definirFecha 10/02/2022
aumentarDias 60
fechaActual
fechaActualExtendida
definirFecha 10/02/2022
aumentarDias 365
fechaActual
fechaActualExtendida
Fin