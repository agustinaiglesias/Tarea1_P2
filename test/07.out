1>Fecha definida en forma exitosa.
2>Se incrementó la fecha.
3>La fecha actual es 11/4/2022.
4>La fecha actual es 11 de Abril de 2022.
5>Fecha definida en forma exitosa.
6>Se incrementó la fecha.
7>La fecha actual es 10/2/2023.
8>La fecha actual es 10 de Febrero de 2023.
9>Fin.
